src/components/atoms/SearchedLocationDisplay/index.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import useRegionQueries from "@/hooks/useRegionQueries";
+import { PageSearchParams } from "@/types/pageSearchParams";
+import { MdLocationOn } from "react-icons/md";
+
+function SearchedLocationDisplay({ searchParams }: PageSearchParams) {
+  const cityId = searchParams?.cityId ? parseInt(searchParams?.cityId, 10) || 0 : 0;
+  const countryId = searchParams?.countryId ? parseInt(searchParams?.countryId, 10) || 0 : 0;
+  const districtId = searchParams?.districtId ? parseInt(searchParams?.districtId, 10) || 0 : 0;
+
+  const queries = useRegionQueries({ cityId, countryId });
+
+  const cities = queries[0]?.data?.data?.response?.cities || [];
+  const countries = queries[1]?.data?.data?.response?.countries || [];
+  const districts = queries[2]?.data?.data?.response?.districts || [];
+
+  const cityName = cities.filter((option: { id: number; name: string }) => option.id === cityId)[0]?.name || "전체";
```

Comment:
find를 사용하시면 0번째 아이템을 꺼내올 필요가 없습니다!
```suggestion
  const cityName = cities.find((option: { id: number; name: string }) => option.id === cityId)?.name || "전체";
```